src/com/houarizegai/calculator/Calculator.java

```diff
@@ -556,21 +556,18 @@ private Calculator() {
     private double calc(double x, String input, char opt) {
         inText.setFont(inText.getFont().deriveFont(Font.BOLD));
         double y = Double.parseDouble(input);
-        if (opt == '+') {
-            return x + y;
-        } else if (opt == '-') {
-            return x - y;
-        } else if (opt == '*') {
-            return x * y;
-        } else if (opt == '/') {
-            return x / y;
-        } else if (opt == '%') {
-            return x % y;
-        } else if (opt == '^') {
-            return Math.pow(x, y);
-        }
         inText.setFont(inText.getFont().deriveFont(Font.PLAIN));
-        return y;
+        switch (opt){
+            case '+': return x + y;
+            case '-': return x - y;
+            case '*': return x * y;
+            case '/': return x / y;
+            case '%': return x % y;
+            case '^': return Math.pow(x , y);
+            default:
+                inText.setFont(inText.getFont().deriveFont(Font.PLAIN));
+                return y;
+        }
```

Comment:
I like this change because it's much better readable, but may you should change to: 

return switch(opt){
    case 'x' -> x + y;
    case '-' -> x - y;
    case '*' -> x * y;

and so on.

Reply:
The requirement of this project is **Java version 8 or higher.**
And the suggested switch statement is available from **JDK 12 Early Access Build 10**

Reply:
Oh, I don't recognised this, sorry.
Then your change is totally fine.

Reply:
It's okay.